### ISSUE TITLE
Panel: color palette doesn't work on light gray background

### ISSUE THREAD

**Issue by bastianallgeier** [getkirby/kirby#39]
Reported by @texnixe

**Comment by jenstornell** [comment]
I had a discussion with @fabianmichael when the color palette was presented.

**Jens**

>They look great as background colors. I hope they will not look too weak as "text" colors. Text colors on a white background often needs to be much stronger to be visible than when used as background colors. Anyway, good work so far.

**Fabian**

>Hey Jens, thanks for you feedback. We started with the colors of tomorrow night, because code samples on our website will have a dark background. But you’re right, the color palette needs to be expanded with darker shades to work on light backgrounds. For things like button backgrounds on the other hand, the current scheme already works very well. Stay tuned for even more colors

As seen here, colors on white needs to be much stronger to work than colors on black:

https://github.com/chriskempson/tomorrow-theme

![image](https://user-images.githubusercontent.com/927011/32263074-6aa62f94-bed8-11e7-9005-af1d0f8df3c5.png)

![image](https://user-images.githubusercontent.com/927011/32263078-704b7274-bed8-11e7-88bc-cf08accac8e2.png)

In conclusion, it's not just about the green, but the palette in general on bright background. If we change the green, we need to change the other colors to keep everything in balance, just as the Tomorrow theme.

***Can we change the title of this issue to "Panel: Implement a color palette that work on bright backgrounds"?***

**Comment by distantnative** [comment]
We added some stronger colors in the last builds.
